beast/tests/test_regresscheck.py

```diff
@@ -42,13 +45,15 @@
     get_lnp_grid_vals,
 )
 from beast.tools.compare_spec_type import compare_spec_type
+from beast.tools.run import create_physicsmodel, create_obsmodel
 
 from beast.tests.helpers import (
     download_rename,
     compare_hdf5,
     compare_tables,
     compare_fits,
 )
+import beast
```

Comment:
Why import all of the `beast` namespace?  We haven't done these anywhere else in the code (that I can think of).  Usually we just import the specific beast module.

Reply:
This is the only way I could figure out to get the path to the module so I could load in the settings file.  But maybe now that it's added to the config file, this isn't necessary any more.  I'll try it and see.

Reply:
If you need the path to the installed location of the beast -> use the `pkg_resources` module.

Reply:
Ah, ok.  I would say that I've never used `pkg_resources` before, but my google history (and #506) tell me that I actually have 😆

Reply:
LOL!